src/System.Diagnostics.Process/src/System/Diagnostics/Process.WinRT.cs

```diff
@@ -13,18 +13,18 @@ private bool StartCore(ProcessStartInfo startInfo)
                 : StartWithCreateProcess(startInfo);
         }
 
-        private string GetMainWindowTitle() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private string GetMainWindowTitle() => string.Empty;
 
-        public IntPtr MainWindowHandle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public IntPtr MainWindowHandle => IntPtr.Zero;
 
-        private bool CloseMainWindowCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool CloseMainWindowCore() => false;
 
-        public string MainWindowTitle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public string MainWindowTitle => string.Empty;
 
-        private bool IsRespondingCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool IsRespondingCore() => true;
 
-        public bool Responding => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public bool Responding => true;
 
-        private bool WaitForInputIdleCore(int milliseconds) => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool WaitForInputIdleCore(int milliseconds) => false;
```

Comment:
Also this pattern is common, which I see in our own tests:
```c#
            using (var process = Process.Start(info)) {
             process.WaitForInputIdle(); // Give the file a chance to load
             // Now do something with process
```
Searching random repos on Github, it seems to be generally used to wait for a process to finish starting in order to eg send keypresses to it.